### PR TITLE
Ix bid adapter: resolve negative size bug

### PR DESCRIPTION
**Pull request by umakajan**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- adding a check to ensure that `BASE_REQ_SIZE < MAX_REQ_SIZE`

### CODE REVIEW

File: modules/ixBidAdapter.js [hunk 1]
```diff
@@ -416,7 +416,11 @@ function buildRequest(validBidRequests, bidderRequest, impressions, version) {
       msd = impressions[transactionIds[i]].missingCount;
     }
 
-    trimImpressions(impressions[transactionIds[i]], MAX_REQ_SIZE - BASE_REQ_SIZE);
+    if (BASE_REQ_SIZE < MAX_REQ_SIZE) {
```
Comment (patmmccann) [inline modules/ixBidAdapter.js:419]: Is this backwards? Trim if req > max ?

**Review by patmmccann — CHANGES_REQUESTED** [review]
Please confirm this logic

File: modules/ixBidAdapter.js (hunk 1 above)
Comment (umakajan) [reply modules/ixBidAdapter.js:419]: We want to make sure that the `BASE_REQ_SIZE` created above isn't larger than the `MAX_REQ_SIZE`, this fix is to ensure that we don't pass a [negative value to `trimImpressions` function](https://github.com/prebid/Prebid.js/pull/6582/files#diff-2597e133de10cd404cf919d82b772fa4a4ba33e49e9f4bdc4ba18a0a905774a5R559).

**Review by patmmccann — APPROVED** [review]
LGTM 

Thanks for confirming the logic, but still in draft state

**Pull request merged**